.github/workflows/build-workflow.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     container:
-      # TODO we will generate it from ODS
+      # TODO https://github.com/opendevstack/ods-quickstarters/issues/766
       image: s2obcn/jdk-11_openj9-wkhtmltopdf-ubi:main

```

Comment:
Could you use version instead main? And could you add the dockerfile used to build the image to the code?

Reply:
Not now

Reply:
Although in a non-standard way (just a Dockerfile in some folder that is not used to build anything), the Dockerfile should be available somewhere.